recipe/common.php

```diff
@@ -46,41 +47,142 @@
  * Configuration
  */
 
-set('keep_releases', 5);
+set('keep_releases', 5, function ($value) {
+    return is_integer($value) && $value >= -1;
```

Comment:
Also will be cool to give users meaningful error message, not just "Validation failed".